### PR TITLE
Develped xyz feature to ease the work.

### PR DESCRIPTION
**Pull request by awsdevopstrainer**

### CODE REVIEW

File: akash.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+welcome to akash file
+
+asd
+fa
+sdf
```
Comment (vijaythuwal) [inline akash.html:5]: hey what did you do here ? its not the right way to do.
change this code

File: santhosh.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+welcome to santhosh file
+
+welcome to online editor
+adsfasdfasdfasdf
+
```
Comment (vijaythuwal) [inline santhosh.html:5]: change this code approach

**Review by vijaythuwal — COMMENTED** [review]
please change the approch

File: akash.html (hunk 1 above)
Comment (awsdevopstrainer) [reply akash.html:5]: this is the correct approach

File: santhosh.html (hunk 2 above)
Comment (awsdevopstrainer) [reply santhosh.html:5]: okay i will change

**Review by vijaythuwal — APPROVED** [review]
its approved

**Pull request merged**